### DISCUSSION TITLE
Dicas de estudo

### DISCUSSION THREAD

**Discussion by gabriel-brito** [frontendbr/forum#1226] [Dicas]
Olá, pessoal! 
Recentemente escrevi um artigo sobre dicas de estudos: [https://medium.com/@mfkrgabriel/algumas-dicas-valiosas-para-estudantes-desesperados-f644313f4ff4](https://medium.com/@mfkrgabriel/algumas-dicas-valiosas-para-estudantes-desesperados-f644313f4ff4)

Quais dicas mais vocês podem compartilhar com outras pessoas?

**Comment by luizbills** [comment]
Dica de ouro, segredo do sucesso: praticar, praticar, praticar e praticar. De preferência praticar com motiva$$ão.

**Comment by ViniciusGularte** [comment]
Acredito muito na gamificação do estudo, para motivar nossos estudos, e poder criar hábitos poderosos.

Vou compartilhar algumas ferramentas que eu uso todos dias de estudo: 

[Habitica](https://habitica.com/) É uma ferramenta open source que eu uso todos dias para gerenciar meus hábitos, ele acaba transformando sua rotina num verdadeiro rpg, pra quem gosta é muito legal.

[HackerRank](https://www.hackerrank.com/) Plataforma de desafio de códigos, sempre uso para melhorar meu raciocínio logico, sem falar que varias empresas utilizam como plataforma de testes para contratação.

[FreeCodeCamp](https://www.freecodecamp.org/) Plataforma mais conhecida, muitos desafios e projetos para fazer por ela, dei uma parada agora, mas usei muito para aprender react, você vai completando os desafios e pode ganhar certificado por eles.

[Duolingo](https://pt.duolingo.com/) Plataforma para aprender outras linguás, uso todos dias, mas mais como um complemento de ensino. Recomendo muito.

**Blogs Geral** Eu ia colocar medium, mas tem muitos outros blogs muito bons por ae, todo dia acordo de manhã, tomo um café e vou ler o que tem de novo, acreditem, sempre tem coisa muito boa por ae de conteúdo. 

Acredito que o lance é praticar, praticar e praticar. Se você for recompensando por isso melhor ainda, a dica do @gabriel-brito  sobre anotar vale muito, tenho utilizado muito no trabalho para expor minhas ideias, e quando tudo é anotado em papeis de rascunho mesmo fica muitoo mais fácil debater acerca disso.
Sobre fazer cursos, eu sempre ia fazendo o que o professor fazia, no final das contas não lembrava de mais nada, comecei a realmente aprender quando pegava os projetos feitos e aprimorava eles, buscando de forma autônoma como fazer melhorias que nem o professor tinha pensado :smile: 

Espero que algumas dessas ferramentas possam ajudar o pessoal :fireworks:

**Comment by linconkusunoki** [comment]
A dica que eu daria é de não focar os estudos em uma ferramenta, mas sim estudar o problema que as ferramentas resolvem, assim fica justificável a existência ou a morte delas. Compreendendo isto fica mais claro perceber para qual caminho a comunidade está seguindo... podendo se antecipar nos estudos, trazer experimentos, novidades e contribuir mais 😄

**Comment by phsantiago** [comment]
Acho legal comentar sobre o modelo 70 20 10 e a importância de errar, que é o que mais atrapalha quando estamos iniciando em algo.
